src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/Licence.kt

```diff
@@ -163,4 +163,14 @@ data class Licence(
       dateCreated = LocalDateTime.now(),
     )
   }
+
+  fun isInPssPeriod(licence: Licence): Boolean {
```

Comment:
now that you have moved this function to licence, you don't need to pass through the licence to the function.

You have access to all this licences fields directly

Reply:
Removed licence param.